kalite/settings/base.py

```diff
@@ -13,8 +15,15 @@
     # Not sure if vars from local_settings are accessed ATM. This is one of
     # the big disadvantage of this whole implicit importing chaos... will be
     # fixed later.
-    from kalite.local_settings import *  # @UnusedWildImport
     from kalite import local_settings
+    # This is not a DeprecationWarning by purpose, because those are
+    # ignored during settings module load time
+    warnings.warn(
+        "Using local_settings is deprecated, please write your own settings "
+        "module, import kalite.settings.base, and point your "
+        "DJANGO_SETTINGS_MODULE to your customized settings or run kalite "
+        "COMMAND --settings=my.settings",
```

Comment:
This might be too terse and cryptic for normal admins, as they may not be aware of all the Django variables and python's import model. Here's a suggestion for the deprecation message:

> We will be deprecating the old way of statically importing custom settings, in favor of a more flexible way. The easiest way to update your installation is to the new way is to add an import statement above your `local_settings.py`:
> `import settings.base`
> and then call `kalite start` with additional arguments: `kalite start --settings=kalite.local_settings`

Some derivative of that. The goal is to have concrete steps for admins to use. Although ideally, the installers would be able to hide a lot of these details behind their shiny interfaces :)